packages/scm/src/browser/scm-widget.tsx

```diff
@@ -94,7 +94,7 @@ export class ScmWidget extends ReactWidget implements StatefulWidget {
         this.toDisposeOnRefresh.dispose();
         this.toDispose.push(this.toDisposeOnRefresh);
         const repository = this.scmService.selectedRepository;
-        this.title.label = repository ? repository.provider.label : 'no repository opened';
+        this.title.label = repository ? repository.provider.label : '';
```

Comment:
it won't work well with multiple views in the scm view container. The header of scm widget part will be empty then.

One can do it by installing https://github.com/microsoft/vscode-extension-samples/tree/master/tree-view-sample from sources and changing its package.json that views are added to the scm view container.